mozilla_django_oidc_db/models.py

```diff
@@ -383,3 +384,188 @@ def get_callback_view(self):
+        default="HS256",
+    )
+    oidc_rp_scopes_list = ArrayField(
+        verbose_name=_("OpenID Connect scopes"),
+        base_field=models.CharField(_("OpenID Connect scope"), max_length=50),
+        default=get_default_scopes,
+        blank=True,
+        help_text=_("OpenID Connect scopes that are requested during login"),
+    )
+    oidc_rp_idp_sign_key = models.CharField(
+        _("Sign key"),
+        max_length=1000,
+        help_text=_(
+            "Key the Identity Provider uses to sign ID tokens in the case of an RSA sign algorithm. "
+            "Should be the signing key in PEM or DER format."
+        ),
+        blank=True,
+    )
+    oidc_token_use_basic_auth = models.BooleanField(
+        _("Use Basic auth for token endpoint"),
+        default=False,
+        help_text=_(
+            "If enabled, the client ID and secret are sent in the HTTP Basic auth "
+            "header when obtaining the access token. Otherwise, they are sent in the "
+            "request body.",
+        ),
+    )
+
+    # Advanced settings
+    oidc_use_nonce = models.BooleanField(
+        _("Use nonce"),
+        help_text=_(
+            "Controls whether the OpenID Connect client uses nonce verification"
+        ),
+        default=True,
+    )
+    oidc_nonce_size = models.PositiveIntegerField(
+        _("Nonce size"),
+        help_text=_(
+            "Sets the length of the random string used for OpenID Connect nonce verification"
+        ),
+        default=32,
+    )
+    oidc_state_size = models.PositiveIntegerField(
+        _("State size"),
+        help_text=_(
+            "Sets the length of the random string used for OpenID Connect state verification"
+        ),
+        default=32,
+    )
+
+    # Keycloak specific config
+    oidc_keycloak_idp_hint = models.CharField(
+        _("Keycloak Identity Provider hint"),
+        max_length=1000,
+        help_text=_(
+            "Specific for Keycloak: parameter that indicates which identity provider "
+            "should be used (therefore skipping the Keycloak login screen)."
+        ),
+        blank=True,
+    )
+
+    userinfo_claims_source = models.CharField(
+        verbose_name=_("user information claims extracted from"),
+        choices=UserInformationClaimsSources.choices,
+        max_length=100,
+        default=UserInformationClaimsSources.userinfo_endpoint,
+        help_text=_(
+            "Indicates the source from which the user information claims should be extracted. This can be the ID token or the User Info endpoint."
+        ),
+    )
+
+    # Customisable options
+    options = JSONField(
+        _("options"),
+        help_text=_("Options relevant for a specific Identity Provider."),
+        schema=OPTIONS_SCHEMA,
+    )
+
+    class Meta:
+        verbose_name = _("OIDC configuration")
+
+    def __str__(self):
+        return f"OIDC Config {self.identifier}"
```

Comment:
```suggestion
    def __str__(self):
        return f"OIDC Config {self.identifier}"
```
please return translatable string from here (`gettext_lazy`)